src/components/FilterDropdown.jsx

```diff
@@ -14,12 +14,13 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 function FilterDropdown(props) {
-  const { label, items, defaultSelect, removeNone } = props;
-  const [choice, setChoice] = useState(`${defaultSelect}`);
+  const { label, items, defaultValue, removeNone, onChange } = props;
+  const [choice, setChoice] = useState(`${defaultValue}`);
```

Comment:
LIke I meant to infer in the other comment, a string template with a string value and no other formatting isn't doing. You can simply do this:

```js
  const [choice, setChoice] = useState(defaultValue);
```

Reply:
ok got it